lib/index.js

```diff
@@ -1,16 +1,98 @@
 'use strict'
 
-var integration = require('@segment/analytics.js-integration')
-
+var integration = require('@segment/analytics.js-integration');
+var useHttps = require('use-https');
 /**
  * Expose `Wigzo` integration
  */
 
 var Wigzo = module.exports = integration('Wigzo')
+  .readyOnInitialize()
+  .global('wigzo')
+  .option('orgToken', '')
+  .option('features', [])
+  .tag('<script src="https://tracker.wigzopush.com/wigzo.js">')
+  .tag('https', '<link rel="manifest" href="/gcm_manifest.json">') /* This needs to be put inside <head>.Is this correct syntax ? */
+  .tag('https', '<script src="https://tracker.wigzopush.com/wigzopush_manager.js?orgtoken={{ orgToken }}">')
+  .tag('http', '<script src="https://tracker.wigzopush.com/gcm_http_subscribe.js?orgtoken={{ orgToken }}">');
 
 /**
  * Initialize Wigzo
  */
 
-Wigzo.prototype.initialize = function () {
-}
+Wigzo.prototype.initialize = function (){
+
+  window.wigzo.setIntegrationSource('Segment');
+
+  window.wigzo.init(settings.orgToken, settings.features);
```

Comment:
can you share docs on what you plan on doing w the `features`? it's default value is an `[]` what did you envision this would look like? what do you need to ask the customer?